React/tic-tac/src/app/screens/Setting/ components/IconSelector/components/Icon/index.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import PropTypes from 'prop-types';
+
+class Icon extends Component {
+  handleClick = () => {
+    this.props.onClick(this.props.name);
+  };
+  render() {
```

Comment:
Add an empty line between methods within a class. 🙂

Reply:
The line break was not reflected